content/ja/getting_started/tracing/_index.md

```diff
@@ -11,7 +11,7 @@ further_reading:
 - link: https://learn.datadoghq.com/courses/intro-to-apm
   tag: ラーニングセンター
   text: Application Performance Monitoring の紹介
-- link: https://dtdg.co/fe
+- link: https://corpsite-preview.datadoghq.com/ccole/tem-lang-pref/technical-enablement/sessions
```

Comment:
preview link

Reply:
Yea thanks for these callouts. I put a note in the PR description to revert these after testing